.changeset/lemon-waves-unite.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/admin.login-flow-builder.v1": patch
+"@wso2is/i18n": patch
+---
```

Comment:
Let's add @wso2is/console package as a patch version update so that a new console version is released with these changes.